packages/client/src/app/challenge/components/ChallengeCheckBox.vue

```diff
@@ -16,7 +16,7 @@
           transform: `rotateY(${deg}deg)`
         }"
       />
-      <p class="position-absolute top-50" :class="buttonClass" :aria-hidden="!checked">
+      <p class="position-absolute top-50" :class="buttonClass" :aria-hidden="true">
```

Comment:
I had changed this to `!checked` so that the `aria-hidden` attribute would change automatically when the `checked` property of the checkmark would change. Why is this reverted?

Reply:
The checkboxes had been disabled for the screen readers with display:none. I have put this back, so that the screen readers now announce the checkbox status (checked/unchecked) of the active boxes (only) instead of the checkmarks. I'd assume this is sufficient, but the checkmarks can be announced too.

The owners checkbox labels are always announced, plus the checkbox status of the active checkboxes.
The opponents checkboxes are not announced as the status has an alternative (the text '2 van de 5 dagen afgerond' gets announced).

Reply:
Ahh, that makes sense